### ISSUE TITLE
[Feature Request]: Allow colormin to take full advantage of colord options

### ISSUE THREAD

**Issue by Irevall** [cssnano/cssnano#1262]
### What should be improved?

Colormin in current state is [just a wrapper](https://github.com/cssnano/cssnano/blob/f93026181d2cf707f308016ffbecab1ceb8aa1d2/packages/postcss-colormin/src/minifyColor.js#L25) for [colord](https://github.com/omgovich/colord) package, but for some reason allows to take control of only 2 out of [6 options](https://github.com/omgovich/colord#color-utilities).
For example for my project I'd like to turn off the `name` transformation, e.g. `#ff0000 -> red`, but I can only change `alphaHex` and `transparent` settings.

### Describe the solution you would like

Instead of passing
```
{
  alphaHex: settings.supportsAlphaHex,
  transparent: settings.supportsTransparent,
  name: true,
}
```
We should pass entire `settings` object with added backwards compatibility  for `supportsAlphaHex` and `supportsTransparent`.

### Are you willing to work on this?

- [X] Yes, I would like to help

**Comment by Irevall** [comment]
Whoops turns out I misread a bit and there is no options support whatsoever. 😅
Added PR.

**Comment by ludofischer** [comment]
Fixed in #1263
